docs/notes/notes.rst

```diff
@@ -5,83 +5,49 @@
 Information about the release lifecycle can be found
 `here <https://github.com/eProsima/Fast-DDS/blob/master/RELEASE_SUPPORT.md>`_.
 
-Version 2.10.4
+Version 2.10.5
 ==============
 
 This release includes the following **features** in an ABI compatible manner:
 
-#. TCP Client and Server Participant Decision Making.
-#. :ref:`Authentication Handshake Properties <property_policies_security>` documentation.
-#. New :ref:`max_message_size property <property_max_message_size>` to limit output datagrams size.
+#. Add XML configuration for :ref:`FlowControllerDescriptor <flowcontrollers_xml>`
 
 This release includes the following **improvements**:
 
-#. Return const reference in ``get_log_resources``.
-#. Include variety of terminate process signals handler in discovery server.
-#. Check History QoS inconsistencies.
-#. Make DataWriters always send the key hash on keyed topics.
-#. ``LARGE_DATA`` Participants logic with same listening ports.
-#. Effectively assert ``AUTOMATIC/MANUAL_BY_PARTICIPANT`` liveliness.
-#. Improve checklist on PR template.
-#. Allow processing of ``AckNack`` submessages with ``count == 0``.
-#. Internal refactor on port handling.
-#. Upgrade Fast CDR submodule to v1.0.28
-
-TCP transport improvements:
-
-#. TCP ``non-blocking`` send.
-#. Enabling multiple interfaces through whitelist in TCP servers.
-#. Set real TCP ``non-blocking-send`` limitation.
-#. Clean up TCP send resources on peer disconnection.
+#. Use `%*` instead of loop in `.bat` scripts
+#. Documentation improvements
+#. Use absolute paths when loading XML files
+#. Reduce tests flakiness
+#. Add unsigned specification to literals
 
 Github CI management:
-
-#. Add macOS Github CI.
-#. Avoid running GitHub CI if PR has conflicts.
-#. Add Ubuntu Github CI.
-#. Improve CI version management.
-#. Pin CMake version and ``vm.mmap_rnd_bits`` in sanitizer workflows.
-#. Only run PRs CI when review is requested.
-#. Refactor Github CI sanitizer related jobs.
-#. Build ``ShapesDemo`` on Ubuntu Github CI.
-#. Fix Python version and environment.
-#. Add DNS entries to hosts files on Github workflows.
-#. Build Fast DDS Python bindings in Fast DDS Docs Github CI job.
+#. Set fallback branch for get_related_branch_from_repo correctly
```

Comment:
I'd add a new line here. I've seen some versions of Sphinx not rendering these lists correctly when they are not separated in a new paragraph.